lib/tasks/gather.js

```diff
@@ -322,6 +322,9 @@ class TaskGather extends SttTask {
     clearTimeout(this.interDigitTimer);
     let resolved = false;
     if (this.dtmfBargein) {
+      if (!this.playComplete) {
```

Comment:
please change to a single condition
```
if (this.dtmfBargein && !this.playComplete) this.notifyStatus..
```

Reply:
added !this.playComplete to the main if condition